site/docs/latest/releaseNotes.md

```diff
@@ -0,0 +1,483 @@
+---
```

Comment:
Can we add all upgrades we did?

Upgrade protobuf to 2.6
Codehale to 3.1.0
ZK to version 3.5
Move to JDK 8
SLF4J to 7.25
Netty 4.1
Guava to version 20.0